packages/kit/test/apps/basics/test/test.js

```diff
@@ -2572,12 +2572,17 @@ test.describe('Static files', () => {
 		expect(response.status()).toBe(process.env.DEV ? 403 : 404);
 	});
 
-	test('Vite serves assets in src directory', async ({ page, request }) => {
+	test('Vite serves assets in allowed directories', async ({ page, request }) => {
 		await page.goto('/assets');
 		const path = await page.textContent('h1');
 
-		const response = await request.get(path);
-		expect(response.status()).toBe(200);
+		const r1 = await request.get(path);
+		expect(r1.status()).toBe(200);
+		expect(await r1.text()).toContain('http://www.w3.org/2000/svg');
+
+		const r2 = await request.get('/package.json');
```

Comment:
this seems surprising to me. I wouldn't expect you to be able to get the `package.json`, `node_modules`, etc. It will also make it available to everyone on your network. I'm not sure that's a huge risk, but I'm not really sure why we'd expose it either

Reply:
It's _not_ getting package.json. The test is there to check that it will serve a route _called_ `/package.json` despite there being a package.json in the project directory outside the `allow` list. Check two lines further down.

node_modules on the other hand are on the allow list so that you can use node_modules!